Files/Views/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -634,12 +639,29 @@
                         <ToolTip Content="{Binding FolderTooltipText}" />
                     </ToolTipService.ToolTip>
                 </controls:DataGridTextColumn>
+                <controls:DataGridTextColumn
+                    x:Name="syncStatusColumn"
+                    x:Uid="syncStatusColumn"
+                    CanUserResize="False"
+                    CanUserSort="False"
+                    Visibility="{x:Bind local:App.CurrentInstance.InstanceViewModel.IsPageTypeOnedrive, Converter={StaticResource BoolToVisibilityConverter}, Mode=OneWay}"
+                    Binding="{Binding SyncStatusGlyph}"
+                    DisplayIndex="2"
+                    FontSize="14"
+                    FontFamily="Segoe MDL2 Assets"
+                    IsReadOnly="True"
+                    Header="Sync"
```

Comment:
```suggestion
                    Header="Status"
```

Reply:
Yup I'll change it in the Resources as well